### PR TITLE
docs(examples): Create example for generate-har.js

### PR DESCRIPTION
**Pull request by anishkny**
Related to #1916

### CODE REVIEW

File: examples/generate-har.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Details at: https://github.com/GoogleChrome/puppeteer/issues/1916
+
+const puppeteer = require('puppeteer');
+const PuppeteerHar = require('puppeteer-har');
```
Comment (aslushnikov) [inline examples/generate-har.js]: let's just add a reference to `puppeteer-har` under the "testing" section in https://github.com/GoogleChrome/puppeteer/tree/master/examples

Comment (anishkny) [reply examples/generate-har.js]: Oh, and not actually add the new file `examples/generate-har.js`?

**Comment by anishkny** [comment]
Removed `examples/generate-har.js` and added link to `puppeteer-har`.

**Review by aslushnikov — APPROVED** [review]
Thanks

**Pull request merged**